dgraph/graphql/src/api.js

```diff
@@ -0,0 +1,34 @@
+import { ApolloServer } from 'apollo-server-fastify';
+import fastify from 'fastify';
+import { typeDefs } from './schema';
+import { dgraph } from './data';
+
+const resolvers = {
+  Query: {
+    searchByName: async (_source, _args, { dataSources }) => {
+      const dgraphQuery = `{
+        query(func: has(code)) @filter(regexp(description, /^${_args.text}.*$/i) AND eq(type, "unit_of_use")) {
```

Comment:
:nit for the regex; it can simplify a little to `/^${_args.text}/i`
Also - the existing query only searches from the start? Pretty sure that's what we'd want anyway, just asking.

Reply:
Thanks for the suggestion r.e. regex. Yeah, this particular query is identical/equivalent to the V1 REST API query which searches from the start (with the current mSupply behaviour I guess it makes sense). 

Thinking with the newer UI we're going to need a few new queries on the GraphQL layer (where possibly searching from the start only may not make as much sense).